networkx/classes/multidigraph.py

```diff
@@ -289,21 +297,30 @@ def add_edge(self, u, v, key=None, **attr):
         multiedge weights.  Convert to Graph using edge attribute
         'weight' to enable weighted graph algorithms.
 
+        Default keys are generated using the method ``new_edge_key(u,v)``.
+        This method can be used elsewhere and/or overloaded if desired.
```

Comment:
Should we say "overridden"?  
Maybe "This method can be overridden by subclassing the base class and providing a custom new_edge_key() method".